assets/airports/cyow.json

```diff
@@ -0,0 +1,1279 @@
+        "THURO": ["N45d33m42.00", "W074d54m47.00"],
+        "TIDUL": ["N45d15m41.00", "W075d21m02.00"],
+        "TILOG": ["N45d09m12.00", "W075d48m52.00"],
+        "TOXIP": ["N45d55m76.00", "W077d04m07.00"],
+        "TUKIR": ["N45d15m12.00", "W076d14m34.00"],
+        "ULAGU": ["N45d11m31.00", "W075d43m66.00"],
+        "URLAG": ["N45d21m55.00", "W075d20m89.00"],
+        "VILRO": ["N45d36m95.00", "W072d42m93.00"],
+        "VINRA": ["N45d16m49.00", "W075d58m57.00"],
+        "VISOL": ["N45d13m34.00", "W075d52m35.00"],
+        "VOLAG": ["N45d08m51.00", "W075d31m75.00"],
+        "WATTO": ["N45d15m97.00", "W075d22m22.00"],
+        "XOLTI": ["N45d28m65.00", "W075d31m52.00"],
+        "YUL"  : ["N45d36m94.00", "W073d58m26.00"]
+    },
+    "runways": [{
+        "name": ["14", "32"],
+        "end": [
+            ["N45d19m37.30", "W075d41m10.40", "140ft"],
+            ["N45d18m38.80", "W075d39m17.70", "367ft"]
+        ],
+        "ils": [false, true]
+    }, {
+        "name": ["07", "25"],
+        "end": [
+            ["N45d18m47.80", "W075d40m16.80", "369ft"],
+            ["N45d19m30.20", "W075d38m42.40", "362ft"]
+        ],
+        "ils": [true, true]
+    }, {
+        "name": ["04", "22"],
+        "end": [
+            ["N45d19m36.00", "W075d41m20.50", "347ft"],
+            ["N45d20m03.10", "W075d41m02.30", "335ft"]
+        ],
+        "ils": [true, true]
+    }],
+    "sids": {
+        "CYOW4": {
+            "icao": "CYOW4",
+            "name": "Ottawa Four",
+            "rwy": {
+                "04": ["#039"],
+                "07": ["#071"],
+                "14": ["#140"],
+                "22": ["#219"],
+                "25": ["#251"],
+                "32": ["#320"]
+            },
+            "exitPoints": {
+                "APLOV": ["APLOV"],
+                "AVVON": ["AVVON"],
+                "IKLAX": ["IKLAX"],
+                "KODEX": ["KODEX"],
+                "LETAK": ["LETAK"],
+                "OLIGO": ["OLIGO"],
+                "RADEN": ["RADEN"],
+                "TUKIR": ["TUKIR"]
+            },
+            "draw": [
+                ["APLOV*"],
+                ["AVVON*"],
+                ["IKLAX*"],
+                ["KODEX*"],
+                ["LETAK*"],
+                ["OLIGO*"],
+                ["RADEN*"],
+                ["TUKIR*"]
+            ]
+        }
+    },
+    "stars": {
+        "CAPTL4": {
+            "icao": "CAPTL4",
+            "name": "Capital Four",
+            "entryPoints": {
+                "MIVOK": ["MIVOK"],
+                "ELSUB": ["ELSUB"]
+            },
+            "body": [["KANIK", "A230"]],
+            "rwy": {
+                "07": [["LANRK", "A100-"], ["VISOL", "A30"], "#071"],
+                "14": [["LANRK", "A160-"], ["VISOL", "A50"], ["SEBEG", "A50"], ["MEGNO", "A35"], "OMTUN", "#320"],
+                "25": [["LANRK", "A160-"], ["VISOL", "A50"], "ULAGU", ["BOREG", "A50|S230"], ["ADVOS", "A30|S210"], "KAREB", "#084"],
```

Comment:
No restriction at VISOL for Runway 25
BOREG should be `["BOREG", "A50|S230-"]`
ADVOS should be `["ADVOS", "A30+|S210-"]`